src/components/TopBar/index.test.ts

```diff
@@ -0,0 +1,7 @@
+import { describe, it, expect } from 'vitest'
+
+describe('Renders main page correctly', async () => {
+  it('Should render the page correctly', async () => {
+    expect(true).equals(true)
+  })
+})
```

Comment:
do we need this test file ?

Reply:
a task will need to be opened to write tests for these components

Both this and the other are just an initial example

Reply:
in my opinion, we remove and when we do the tests, we create. But we can go on like this